### ISSUE TITLE
Calculating new metadata for regression fails for most datasets

### ISSUE THREAD

**Issue by he-ritter** [automl/auto-sklearn#840]
Hello,

When calculating new metadata for regression, a flurry of error messages similar to the following one occur:

```
Traceback (most recent call last):
  File "/XXX/autosk_reg/scripts/run_auto-sklearn_for_metadata_generation.py", line 62, in <module>
    X_train, y_train, X_test, y_test, cat = load_task(task_id)
  File "/XXX/autosk_reg/scripts/update_metadata_util.py", line 41, in load_task
    y_test = np.array([mapping[value] for value in y_test])
  File "/XXX/autosk_reg/scripts/update_metadata_util.py", line 41, in <listcomp>
    y_test = np.array([mapping[value] for value in y_test])
KeyError: 8.822999954223633

```

Only the keys are different for each error.

I think that this then also crashes the processing of the dataset, as the only datasets that are actually being processed are 5022 and 5024.

**Comment by mfeurer** [comment]
Thanks for reporting this issue. It appears that we broke the code for metadata generation for regression datasets. This mapping should only be constructed for classification.

I'd be happy about a pull request circumventing this check.

**Comment by mfeurer** [comment]
Will be fixed in #951

**Comment by mfeurer** [comment]
This is fixed via #951 and we added regression metadata in ef5685e1c1ac61f0267ee1f504c92b6c867a3ce7. This will be available in the next release.
